pkg/cmd/cli/cmd/wrappers.go

```diff
@@ -310,29 +310,29 @@ foreground for an interactive container execution.  You may pass 'run/v1' to
 --generator to create a replication controller instead of a deployment config.`
 
 	runExample = `  # Starts a single instance of nginx.
-  $ %[1]s run nginx --image=nginx
+  %[1]s run nginx --image=nginx
 
   # Starts a replicated instance of nginx.
-  $ %[1]s run nginx --image=nginx --replicas=5
+  %[1]s run nginx --image=nginx --replicas=5
 
   # Dry run. Print the corresponding API objects without creating them.
-  $ %[1]s run nginx --image=nginx --dry-run
+  %[1]s run nginx --image=nginx --dry-run
 
   # Start a single instance of nginx, but overload the spec of the replication
   # controller with a partial set of values parsed from JSON.
-  $ %[1]s run nginx --image=nginx --overrides='{ "apiVersion": "v1", "spec": { ... } }'
+  %[1]s run nginx --image=nginx --overrides='{ "apiVersion": "v1", "spec": { ... } }'
 
   # Start a single instance of nginx and keep it in the foreground, don't restart it if it exits.
-  $ %[1]s run -i --tty nginx --image=nginx --restart=Never`
+  %[1]s run -i --tty nginx --image=nginx --restart=Never`
 
 	// TODO: uncomment these when arguments are delivered upstream
```

Comment:
Does anybody know if this should be uncommented? The TODO was added in https://github.com/openshift/origin/pull/4299, last August.